packages/devtools_app/lib/src/service_manager.dart

```diff
@@ -380,6 +384,20 @@ class IsolateManager {
 
   Stream<IsolateRef> get onIsolateExited => _isolateExitedController.stream;
 
+  /// Return a unique, monotonically increasing number for this Isolate.
+  int isolateIndex(IsolateRef isolateRef) {
+    if (!_isolateIndexMap.containsKey(isolateRef.id)) {
```

Comment:
this check happens here and in _assignIsolateIndex. Can we remove this one and rename _assignIsolateIndex to _maybeAssignIsolateIndex or similar?

Reply:
done - just used the existing `isolateIndex()`